src/components/login/index.js

```diff
@@ -50,15 +51,16 @@ const LoginComponent = ({
                         {title}
                         <div className={styles.input}>
                             <input placeholder="youremail@example.com" value={email} onChange={e => setEmail(e.target.value)}
-                                   onKeyPress={(ev) => ev.key === 'Enter' ? loginCode() : null} data-testid="email-input" />
-                            <button onClick={() => loginCode()} data-testid="email-button">
-                                &gt;
-                            </button>
-                            <br />
+                                   onKeyPress={(ev) => ev.key === 'Enter' ? loginCode() : null} data-testid="email-input" />                           
```

Comment:
@syoungfntech i dont see the HTML for the summit image if its present

Reply:
Per the figma the summit logo and primary color for the button should change. 
<img width="807" alt="image" src="https://github.com/fntechgit/summit-registration-lite/assets/104139/0611f2a4-3557-49b5-81fc-06e3a43e8f76">

Reply:
Looks like the summit logo is different from the one in Figma. I'm guessing this will be fixed via uploading the correct logo in digitalocean

Reply:
Also, I had to pass in summitData to LoginComponent from @openeventkit/event-site to get the data to show up here.